lib/features/auth/ui/sign_in.dart

```diff
@@ -53,7 +56,38 @@ class SignInScreen extends StatelessWidget {
                   mainAxisAlignment: MainAxisAlignment.center,
                   children: [
                     const Spacer(),
-                    const Image(image: AssetImage(AppAssets.flutterConKeLogo)),
+                    GestureDetector(
+                      onLongPress: () {
+                        final config = getIt<FirebaseRepository>().getConfig();
+                        if (config.isInReview &&
+                            config.appVersion == Misc.getAppVersion()) {
```

Comment:
Does this mean config.appversion will be the version of the app that's in review?

Reply:
Yes yes. That way there's no possibility of a leak with backward versions. 

The config to control this as well is on firebase remote config.